sacrebleu/metrics/ter.py

```diff
@@ -130,14 +130,15 @@ def corpus_score(self, sys_stream: Union[str, Iterable[str]],
 
             words_hyp = self.tokenizer(hypo).split()
 
-            best_num_edits = None
+            # `None` init made mypy unhappy. Make it relatively large at start
+            best_num_edits = int(1e16)
```

Comment:
Does this sound OK @ales-t ? `mypy` does not like a None variable being assigned something else dynamically.

Reply:
Looks good!